zk_toolbox/crates/zk_inception/src/commands/contract/build.rs

```diff
@@ -0,0 +1,18 @@
+use common::{cmd::Cmd, spinner::Spinner};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use crate::messages::MSG_CONTRACT_BUILDING;
+
+pub(crate) async fn run(shell: &Shell) -> anyhow::Result<()> {
+    let ecosystem = EcosystemConfig::from_file(shell)?;
+
+    let _dir_guard = shell.push_dir(&ecosystem.link_to_code);
+    let spinner = Spinner::new(MSG_CONTRACT_BUILDING);
+
+    Cmd::new(cmd!(shell, "yarn l1-contracts build")).run()?;
```

Comment:
we don't need to build l1 contracts we use forge for it

Reply:
Removed in https://github.com/matter-labs/zksync-era/pull/2515/commits/d42c7604f48ba1633f2b1760d0d620c507cfd45b